drivers/pinctrl/pinctrl_sy1xx.c

```diff
@@ -0,0 +1,63 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright (c) 2024 sensry.io
+ */
+
+#define DT_DRV_COMPAT sensry_sy1xx_pinctrl
+
+#include <zephyr/drivers/pinctrl.h>
+
+#include <zephyr/arch/common/sys_io.h>
+
+#define SY1XX_PAD_CONFIG_BASE_ADDR      (0x1A104000)
```

Comment:
why isn't this in DT?

Reply:
good point. can be done.

Reply:
actually there need to be a context for pinctrl to pull in the address from DT. not sure if there is a pinctrl context.

Reply:
solved, from my perspective.